### ISSUE TITLE
difference between the two species records?

### ISSUE THREAD

**Issue by myrmoteras** [gbif/portal-feedback#5097]
what is the difference here?

![image](https://github.com/gbif/portal-feedback/assets/4609956/0aeb5ae0-8f88-452e-856a-4b862afc28d4)

https://www.gbif.org/species/3218379  

https://www.gbif.org/species/209566722   

both are `species`, so what is there difference? The latter 722 is cited by the former as treatment.

@daniel-mietchen @mdoering

**Comment by ManonGros** [comment]
@myrmoteras 

https://www.gbif.org/species/3218379 is the entry corresponding to the GBIF Backbone taxonomy. All the occurrences and checklist taxon records with this name are matched to this entry to make this page. There are a lot of aggregated data: occurrence images, occurrence map, PLAZI treatment, checklist description, occurrence corresponding to type specimens, introduced and invasive status, etc.

https://www.gbif.org/species/209566722 is the taxon record from the checklist titled [First report of diversity of Cyanobacteria of Broknes Peninsula of Larsemann Hills, East Antarctica](https://www.gbif.org/dataset/fe7e93a2-0211-4f12-8bb4-aaf287f1f853). It contains the information made available in the checklist and a link to the GBIF Backbone taxonomy (when available).

**Comment by myrmoteras** [comment]
thanks, @ManonGros 
What are the criteria to match? just the genus and species epithet, or also the authority? Is the match fuzzy, eg including rufa, rufus or just the version in the GBIF backbone?

Who is making the link to the GBIF Backbone taxonomy - the matching service, or editors - and if so, who would that be?

**Comment by mdoering** [comment]
That is done automatically whenever datasets are imported using the backbone matching service, just like we do for occurrences. But it uses a slightly different algorithm that is not fuzzy on the name, just for the authorship

**Comment by mdoering** [comment]
A similar thing happens on checklistbank.org (CLB) btw, just that there the matching is done automatically to the "Names Index" which contains all unique names in CLB instead of matching to one. specific dataset only (the backbone) which causes problems for names not in the backbone (yet)

**Comment by myrmoteras** [comment]
that means [Cataglyphis niger](https://www.gbif.org/species/1319641) is in the GBIF taxonomic backbone not the same as [Cataglyphis nigra](https://www.gbif.org/species/10880081)? 

se the ending - er vs -ra

**Comment by mdoering** [comment]
Yes, that is right. Unfortunately so. Note that C.nigra is even listed as a synonym for Messor capitatus.
In Checklistbank you can see the improvements in using better stemming: https://www.checklistbank.org/namesindex/4195331/group
